README.md

```diff
@@ -73,3 +73,20 @@ vgrep supports the following commands:
 
 ## Files
 ![](screenshots/vgrep-files.png)
+
+## fzf
+
+![](https://user-images.githubusercontent.com/7258858/103111382-b00c7f80-464c-11eb-9e47-c36ed89253a1.png)
+
+Running vgrep twice to open an editor is not ideal. The below function uses [fzf](https://github.com/junegunn/fzf) to iteratively search with vgrep and open your editor with one enter at the correct line.
```

Comment:
I don't like the first sentence.  It really depends on the intended work flow.  We've discussed implementing a pager in vgrep in some issue, I believe in 2019, but I do not want to go down this road.  It's hard to get interactive UIs right and to fit the various work flows people have.  In this context, using `fzf` is a super cool way to get there.

I suggest to rewrite and explain what `fzf` is, and how it can enhance vgrep for the specific work flow.

Reply:
> I don't like the first sentence.

Do you have something in mind that you like? I have no idea.

> I suggest to rewrite and explain what fzf is

I think everyone should know fzf and if not I linked the repo where you can find out about it. I can't explain what fzf is in a few short sentences that are easy to understand because the tool is simply to powerful and does to much on its own.

> and how it can enhance vgrep for the specific work flow.

I explained that already.

```suggestion
Running vgrep twice to open an editor is not ideal. The below function uses [fzf](https://github.com/junegunn/fzf) to interatively search with vgrep and open your editor with one enter at the correct line.
```

Reply:
> > I don't like the first sentence.
> 
> Do you have something in mind that you like? I have no idea.

Yes, as mentioned, I would prefer to explain what fzf is. "Not ideal" does not add information. "If you desire a more interactive experience than running `vgrep` twice to first search and another time to open an editor, you may have a look at `fzf`.", for example, may be more expressive.
 
> > I suggest to rewrite and explain what fzf is
> 
> I think everyone should know fzf and if not I linked the repo where you can find out about it. I can't explain what fzf is in a few short sentences that are easy to understand because the tool is simply to powerful and does to much on its own.

I have never heard of `fzf` until this pull request and do not assume that everyone does.  Linking to the GitHub page of `fzf` does not help me (at least) much as there's way too much information to understand how `fzf` and `vgrep` could interact.

Reply:
```suggestion
If you desire a more interactive experience than running vgrep twice to first search and then to open an editor, you may have a look at fzf. The below function uses [fzf](https://github.com/junegunn/fzf) to iteratively search with vgrep and open your editor with one enter at the correct line.
```

Reply:
s/interatively/interactively/

Besides that, LGTM. Thanks!